Kajabi/sage-lib
# DSS-481: fix(hero): Update large hero max width to full

## Description
In some cases, the large hero does not expand to the full width of its container. This appears to be due to an incorrect max-width being set on the large hero.

This PR will update the max-width to full.


## Screenshots

| |  Before  |  After  |
|--------|--------|--------|
| Docs | <img width="1207" alt="Screenshot 2023-10-23 at 9 20 15 AM" src="https://github.com/Kajabi/sage-lib/assets/1175111/dd8a2445-5bfa-4224-9d0d-cf0742806503">|<img width="1215" alt="Screenshot 2023-10-23 at 9 20 35 AM" src="https://github.com/Kajabi/sage-lib/assets/1175111/168cedfc-0c72-4e21-8659-570fbf35a66f">|
| KP |<img width="1287" alt="Screenshot 2023-10-23 at 9 21 24 AM" src="https://github.com/Kajabi/sage-lib/assets/1175111/4f4cf200-9b71-4113-a5b7-2e6edb609758">|<img width="1421" alt="Screenshot 2023-10-23 at 9 21 00 AM" src="https://github.com/Kajabi/sage-lib/assets/1175111/f641d01a-31c8-4c73-bc1f-dd1a0d5a1e52">|

## Testing in `sage-lib`

- Navigate to [Hero](http://localhost:4000/pages/component/hero?tab=preview)
- Verify Large Hero Expands to Container

## Testing in `kajabi-products`

1. (**LOW**) Updates max-width of large hero component. The hero should expand to the full width of its container.
   - [ ] Dashboard Hero
   - [ ] Creator Studio Hero


## Related
DSS-481
